rpcd/playbooks/roles/elasticsearch/tasks/upgrade_elasticsearch_pre.yml

```diff
@@ -0,0 +1,256 @@
+  tags:
+    - logging-upgrade
+    - elasticsearch-upgrade
+    - reindex-wrapper
+
+- name: Remove legacy liberty template script
+  file:
+    name: '/opt/legacy-mapping.sh'
+    state: absent
+  delegate_to: "{{ groups['elasticsearch_all'][0] }}"
+  tags:
+    - logging-upgrade
+    - elasticsearch-upgrade
+    - reindex-wrapper
+
+- name: Install Reindexing Plugin
+  command: ./plugin -install {{ item }}
+  args:
+    chdir: /usr/share/elasticsearch/bin
+    creates: /usr/share/elasticsearch/plugins/reindexing
+  with_items:
+    - org.codelibs/elasticsearch-reindexing/1.7.0
+  tags:
+    - logging-upgrade
+    - elasticsearch-upgrade
+    - reindex-wrapper
+
+- name: Stop elasticsearch
+  service:
+    name: elasticsearch
+    state: stopped
+  tags:
+    - logging-upgrade
+    - elasticsearch-upgrade
+    - reindex-wrapper
+
+- name: Make sure elasticsearch is stopped
+  command: 'pkill -f elasticsearch'
+  ignore_errors: true
+  tags:
+    - logging-upgrade
+    - elasticsearch-upgrade
+    - reindex-wrapper
+
+- name: Start elasticsearch
+  service:
+    name: elasticsearch
+    state: started
+  tags:
+    - logging-upgrade
+    - elasticsearch-upgrade
+    - reindex-wrapper
+
+- name: Wait for elasticsearch to restart
+  wait_for:
+    host: "{{ hostvars[groups['elasticsearch_container'][0]]['container_address'] }}"
+    port: 9200
+    state: started
+    timeout: 300
+  tags:
+    - logging-upgrade
+    - elasticsearch-upgrade
+    - reindex-wrapper
+
+- name: Install reindexing wrapper script
+  copy:
+    src: reindex-liberty.py
+    dest: /opt/reindex-liberty.py
+    owner: root
+    group: root
+    mode: 0755
+  delegate_to: "{{ groups['elasticsearch_all'][0] }}"
+  tags:
+    - logging-upgrade
+    - elasticsearch-upgrade
+    - elasticsearch-reindex
+    - reindex-wrapper
+
+- name: Reindex legacy elasticsearch indices
+  shell: /opt/reindex-liberty.py --host "{{ hostvars[groups['elasticsearch_container'][0]]['container_address'] }}" --reindex
+  delegate_to: "{{ groups['elasticsearch_all'][0] }}"
+  tags:
+    - logging-upgrade
+    - elasticsearch-upgrade
+    - elasticsearch-reindex
+
+- name: Monitor reindexing process
+  shell: /opt/reindex-liberty.py --host "{{ hostvars[groups['elasticsearch_container'][0]]['container_address'] }}" --monitor --continuous
+  async: 259200
+  poll: 5
+  delegate_to: "{{ groups['elasticsearch_all'][0] }}"
+  tags:
+    - logging-upgrade
+    - elasticsearch-upgrade
+    - elasticsearch-reindex
+
+- name: Drop legacy elasticsearch indeices
+  shell: /opt/reindex-liberty.py --host "{{ hostvars[groups['elasticsearch_container'][0]]['container_address'] }}" --drop
+  delegate_to: "{{ groups['elasticsearch_all'][0] }}"
+  tags:
+    - logging-upgrade
+    - elasticsearch-upgrade
+    - elasticsearch-reindex
+
+- name: Remove Reindexing Plugin
+  command: ./plugin --remove {{ item }}
+  args:
+    chdir: /usr/share/elasticsearch/bin
+    creates: /usr/share/elasticsearch/plugins/reindexing
```

Comment:
By the looks of it, this creates argument is preventing the command running. That path is expected to already exist and should be removed by this step.

I think either creates should be changed to removes or it should not be used at all.